home_work_2_zd_4.js

```diff
@@ -0,0 +1,10 @@
+let newStr = "";
+function concat(str) {
```

Comment:
this is not a closure realization. Need rewrite. Don't use global scope

Reply:
Done